safehttp/mux.go

```diff
@@ -170,5 +170,7 @@ func (m methodHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
-	h.ServeHTTP(rw, ir)
+	if res := h.ServeHTTP(rw, ir); !res.written {
+		rw.NoContent()
```

Comment:
The go default behavior is to return a 200 OK with no body, maybe let's use that?

Reply:
Right, makes sense. But how?
```go
rw.Write(nil) // nil must be a valid safe response in the dispatcher
```
```go
rw.rw.WriteHeader(200) // sidestep dispatcher just like in NoContent
```
Or add a new `WriteNoContent` method?

Reply:
I would WriteHeader(200) or do nothing. The http package will take care of setting a 200.

Reply:
Sorry, I missed your response. But I opened #153 so this won't be forgotten.